extension/sumologicextension/extension.go

```diff
@@ -700,7 +700,7 @@ var sumoAppProcesses = map[string]string{
 	"apache":                "apache",
 	"apache2":               "apache",
 	"httpd":                 "apache",
-	"docker":                "docker",
+	"docker":                "docker", // docker cli
```

Comment:
this wasn't mentioned but I believe this is for docker cli only and not docker desktop, since one of the new products being added is called ```docker ce```.